build.xml

```
@@ -1857,32 +1864,50 @@
+      <!-- Support Simulator Tests -->
+    <string>-javaagent:${test.lib}/jars/simulator-asm.jar</string>
+    <string>-Xbootclasspath/a:${test.lib}/jars/simulator-bootstrap.jar</string>
+    <string>-XX:ActiveProcessorCount=4</string>
+    <string>-XX:-TieredCompilation</string>
+    <string>-XX:-BackgroundCompilation</string>
+    <string>-XX:CICompilerCount=1</string>
+    <string>-XX:Tier4CompileThreshold=1000</string>
+    <string>-XX:ReservedCodeCacheSize=256M</string>
+    <!-- total memory must fit within the pod constraints, see comments in .jenkins/Jenkinsfile and dind's container resourceRequestMemory in .jenkins/k8s/jenkins-deployment.yaml -->
+    <string>-Xmx8G</string>
+    <!-- Harry tests kept failing due to direct memory failures and looks like its undersized... so upping to allow more stable runs -->
+    <string>-XX:MaxDirectMemorySize=8G</string>
+  </resources>
+  <pathconvert property="_simulator.jvmargs_concat" refid="_simulator_jvmargs_items" pathsep=" "/>
+
   <target name="test-simulator-dtest" depends="maybe-build-test" description="Execute simulator dtests">
     <testmacro inputdir="${test.simulator-test.src}" timeout="${test.simulation.timeout}" forkmode="perTest" showoutput="true" filter="**/test/${test.name}.java" maxmemory="8g">
-      <jvmarg value="-Dlogback.configurationFile=test/conf/logback-simulator.xml"/>
-      <jvmarg value="-Dcassandra.ring_delay_ms=10000"/>
-      <jvmarg value="-Dcassandra.tolerate_sstable_size=true"/>
-      <jvmarg value="-Dcassandra.skip_sync=true" />
-      <jvmarg value="-Dcassandra.debugrefcount=false"/>
-      <jvmarg value="-Dcassandra.test.simulator.determinismcheck=strict"/>
-      <jvmarg value="-Dcassandra.test.simulator.print_asm=${simulator.asm.print}" />
-      <!-- Support Simulator Tests -->
-      <jvmarg line="-javaagent:${test.lib}/jars/simulator-asm.jar"/>
-      <jvmarg line="-Xbootclasspath/a:${test.lib}/jars/simulator-bootstrap.jar"/>
-      <jvmarg line="-XX:ActiveProcessorCount=4"/>
-      <jvmarg line="-XX:-TieredCompilation"/>
-      <jvmarg line="-XX:-BackgroundCompilation"/>
-      <jvmarg line="-XX:CICompilerCount=1"/>
-      <jvmarg line="-XX:Tier4CompileThreshold=1000"/>
-      <jvmarg line="-XX:ReservedCodeCacheSize=256M"/>
-      <!-- total memory must fit within the pod constraints, see comments in .jenkins/Jenkinsfile and dind's container resourceRequestMemory in .jenkins/k8s/jenkins-deployment.yaml -->
-      <jvmarg line="-Xmx8G"/>
-      <!-- Harry tests kept failing due to direct memory failures and looks like its undersized... so upping to allow more stable runs -->
-      <jvmarg line="-XX:MaxDirectMemorySize=8G"/>
+      <jvmarg line="${_simulator.jvmargs_concat}"/>
     </testmacro>
   </target>
 
+  <macrodef name="testlist-simulator">
+    <attribute name="test.file.list" />
+    <sequential>
+      <testmacrohelper inputdir="${test.simulator-test.src}" timeout="${test.timeout}"
```

```suggestion
      <testmacrohelper inputdir="${test.simulator-test.src}" timeout="${test.simulator.timeout}"
```
Should this be simulator timeout instead?

---

it is intentional, to follow the style of other testlist definitions, the timeout is propagated from run-tests.sh logic (_run_testlist)